core/src/types.ts

```diff
@@ -205,4 +214,70 @@ export declare interface SurplusTransactionFees {
     combinedBidFeesDai: BigNumber;
 }
 
-export declare interface SurplusAuctionTransaction extends SurplusAuctionEnriched, SurplusTransactionFees {}
+export declare interface SurplusAuctionTransaction
+    extends SurplusAuctionEnriched,
+        CompensationAuctionTransactionFees {}
+
+export declare interface DebtAuctionBase {
+    id: number;
+    network: string;
+    fetchedAt: Date;
+}
+
+export declare interface DebtAuctionActive extends DebtAuctionBase {
+    bidAmountDai: BigNumber;
+    receiveAmountMKR: BigNumber;
+    receiverAddress: string;
+    auctionEndDate: Date;
+    bidEndDate?: Date;
+    earliestEndDate: Date;
+    state: CompensationAuctionActiveState;
+}
+
+export declare interface DebtAuctionCollected extends DebtAuctionBase {
+    state: CompensationAuctionInactiveState;
+}
+
+export type DebtAuction = DebtAuctionActive | DebtAuctionCollected;
+
+export type DebtAuctionStates = DebtAuction['state'];
+
+export type DebtAuctionActionStates = 'loaded' | 'restarting' | 'bidding' | 'collecting';
+
+export declare interface DebtAuctionEnriched extends DebtAuctionActive {
+    nextMaximumLotReceived: BigNumber;
+    marketUnitPrice: BigNumber | string;
+    marketUnitPriceToUnitPriceRatio: BigNumber;
+    unitPrice: BigNumber;
+}
+
+export declare interface DebtAuctionTransaction extends DebtAuctionEnriched, CompensationAuctionTransactionFees {}
+
+export const getCompensationAuctionTransactionFees = async function (
```

Comment:
I assume this is in here for testing right? We should remove it tho, as we only want to sort out the typing in this PR.